ovos_ww_plugin_vosk/__init__.py

```
@@ -168,9 +169,9 @@
     SEC_BETWEEN_WW_CHECKS = 0.2
     MAX_EXPECTED_DURATION = 3  # seconds of data chunks received at a time
 
-    def __init__(self, hotword="hey mycroft", config=None, lang="en-us"):
+    def __init__(self, hotword="hey mycroft", config=None):
         config = config or {}
-        super(VoskWakeWordPlugin, self).__init__(hotword, config, lang)
+        super(VoskWakeWordPlugin, self).__init__(hotword, config)
```

Pylint (3.3.7)

[refactor] Consider using Python 3 style super() without arguments

(R1725)